### PR TITLE
TrackballControls: Fix scale finished rotate on mobile.

### PR DESCRIPTION
**Pull request by leoshenhh**
**Description**

fix TrackballControls.js  scale finished rorate on mobild with two finger

### CODE REVIEW

File: examples/js/controls/TrackballControls.js [hunk 1]
```diff
@@ -687,6 +687,16 @@
 
 						break;
 
+					case 2:
```
Comment (Mugen87) [inline examples/js/controls/TrackballControls.js:690]: Please always edit files in the `examples/jsm` directory. `examples/js` is generated.

Comment (leoshenhh) [reply examples/js/controls/TrackballControls.js:690]: Thanks for the reminder !
Should I submit another pr ?

Comment (Mugen87) [reply examples/js/controls/TrackballControls.js]: Not necessary.

**Comment by Mugen87** [comment]
https://raw.githack.com/leoshenhh/three.js/dev/examples/misc_controls_trackball.html

File: examples/jsm/controls/TrackballControls.js [hunk 2]
```diff
@@ -706,6 +706,12 @@ class TrackballControls extends EventDispatcher {
 					_movePrev.copy( _moveCurr );
 					break;
 
+				case 2:
```
Comment (Mugen87) [inline examples/jsm/controls/TrackballControls.js:709]: Can you please describe in more detail why you have added this section? 

AFAICS, this is only relevant when three pointers were active and one of them was released. However, `TrackballControls` does not make use of more than two pointers. So what does this code exactly do?

Comment (leoshenhh) [reply examples/jsm/controls/TrackballControls.js:709]: In my actual use that I found ,`case 2` will be triggered when I take away the one of tow pointers;
And` case 0` never be triggered

Comment (leoshenhh) [reply examples/jsm/controls/TrackballControls.js:709]: this is my step on mobile
Step one: I zoom objcet with two pointer
Step two: I release one of them pointer
Step three:  I move left pointer then the object will rotate just like one pointer rorate  before but not zoom
I think it shouldn't be like that

Comment (Mugen87) [reply examples/jsm/controls/TrackballControls.js:709]: > Step one: I zoom objcet with two pointer
Step two: I release one of them pointer
Step three: I move left pointer then the object will rotate just like one pointer rorate before but not zoom

Um, but this sounds like the expected behavior. The state of the controls corresponds to the number of active pointers.

Comment (leoshenhh) [reply examples/jsm/controls/TrackballControls.js:709]: But in step three I just move a little distance but the object rotate very much!

Comment (leoshenhh) [reply examples/jsm/controls/TrackballControls.js:709]: And I continue move pointer , the object rotate will be normal,
It just happend in zoom finished once that rotate very much 
Buy if you release tow pointers same time ,it won't happen

Comment (leoshenhh) [reply examples/jsm/controls/TrackballControls.js:709]: reference  #7406
but it's not fix this bug complete

Comment (Mugen87) [reply examples/jsm/controls/TrackballControls.js:709]: > And I continue move pointer , the object rotate will be normal,
It just happend in zoom finished once that rotate very much
Buy if you release tow pointers same time ,it won't happen

That cleared things up. Indeed, the "jump" does not appear with your version.

**Review by Mugen87 — APPROVED** [review]

File: examples/jsm/controls/TrackballControls.js (hunk 2 above)
Comment (leoshenhh) [reply examples/jsm/controls/TrackballControls.js:709]: Thanks man，so my pr will be merge？

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**